persist-cli/src/main/java/io/ballerina/persist/nodegenerator/BalSyntaxGenerator.java

```diff
@@ -1054,4 +1058,11 @@ private static String getEntityNameConstant(String entityName) {
         }
         return outputString.toString();
     }
+
+    private static String formatFieldName(String fieldName) {
+        if (fieldName.startsWith("\'")) {
+            return fieldName.substring(1);
+        }
+        return fieldName;
```

Comment:
```suggestion
        return fieldName.startsWith("'") ? fieldName.substring(1) : fieldName;
```

Reply:
updated in https://github.com/ballerina-platform/persist-tools/pull/117/commits/06860b7f7fa6a73ba0dfad04d67d653c21f63dd1